### PR TITLE
Multiple decorators - fix cases with include/exclude plugins

### PR DESCRIPTION
**Pull request by jonatan1609**

### CODE REVIEW

**Comment by jonatan1609** [comment]
#641

File: pyrogram/client.py [hunk 1]
```diff
@@ -749,13 +749,12 @@ def load_plugins(self):
...
+                            for handler, group in getattr(module, name).handlers:
+                                if isinstance(handler, Handler) and isinstance(group, int):
+                                    self.add_handler(handler, group)
 
-                                log.info('[{}] [LOAD] {}("{}") in group {} from "{}"'.format(
-                                    self.session_name, type(handler).__name__, name, group, module_path))
+                                    log.info('[{}] [LOAD] {}("{}") in group {} from "{}"'.format(
+                                        self.session_name, type(handler).__name__, name, group, module_path))
 
                                 count += 1
```
Comment (delivrance) [inline pyrogram/client.py:759]: The indentation here is still wrong; `count += 1` will also count anything that's not an handler.

File: pyrogram/client.py [hunk 2]
```diff
@@ -785,13 +784,12 @@ def load_plugins(self):
...
+                            for handler, group in getattr(module, name).handlers:
+                                if isinstance(handler, Handler) and isinstance(group, int):
+                                    self.remove_handler(handler, group)
 
-                                log.info('[{}] [UNLOAD] {}("{}") from group {} in "{}"'.format(
-                                    self.session_name, type(handler).__name__, name, group, module_path))
+                                    log.info('[{}] [UNLOAD] {}("{}") from group {} in "{}"'.format(
+                                        self.session_name, type(handler).__name__, name, group, module_path))
 
                                 count -= 1
```
Comment (delivrance) [inline pyrogram/client.py:794]: Same here.

**Review by delivrance — CHANGES_REQUESTED** [review]

File: pyrogram/client.py (hunk 2 above)
Comment (jonatan1609) [reply pyrogram/client.py]: fixed

File: pyrogram/client.py (hunk 1 above)
Comment (jonatan1609) [reply pyrogram/client.py]: oh wait, logging

Comment (jonatan1609) [reply pyrogram/client.py]: i think it is solved, in wonder why in this Change Request you marked the logging lines

**Comment by jonatan1609** [comment]
i don't know why you requested changing in those logging lines, I changed and pushed the counting.

File: pyrogram/client.py (hunk 1 above)
Comment (delivrance) [reply pyrogram/client.py]: No, I didn't, I selected the line containing `count += 1`, but you get a preview of the above code as well.

**Comment by delivrance** [comment]
I did not. I selected the line containing `count += 1`, but you get a preview of the above code as well.

File: pyrogram/client.py (hunk 1 above)
Comment (jonatan1609) [reply pyrogram/client.py]: ah, that's confusing 
image

**Review by delivrance — APPROVED** [review]

**Pull request merged**